examples/02_first_level_models/plot_predictions_residuals.py

```diff
@@ -0,0 +1,121 @@
+"""
+Predicted time series and residuals
+===================================
+ 
+Here we fit a First Level GLM with the `minimize_memory`-argument set to `False`.
+By doing so, the `FirstLevelModel`-object stores the residuals, which we can then inspect.
+Also, the predicted time series can be extracted, which are useful to assess the quality of the model fit.
+"""
+
+
+#########################################################################
+# Import modules
+# --------------
+from nistats.datasets import fetch_spm_auditory
+from nilearn.plotting import plot_stat_map
+from nilearn import input_data, image
+import matplotlib.pyplot as plt
+from nilearn import plotting, masking
+from nistats.reporting import get_clusters_table
+from nistats.first_level_model import FirstLevelModel
+import pandas as pd
+
+# load fMRI data
+subject_data = fetch_spm_auditory()
+fmri_img = image.concat_imgs(subject_data.func)
+
+# Make an average
+mean_img = image.mean_img(fmri_img)
+mask = masking.compute_epi_mask(mean_img)
+
+# Clean (demean) and smooth data
+fmri_img = image.clean_img(fmri_img)
```

Comment:
It is not a very good idea to demean fMRI time series, as people like to express the estimated signal as percent of the baseline change.  But, for this, you need to know the baseline signal.

Somebody complained about the signal being rescaled, this is actually the reason.

Reply:
I think I've addressed this by changing `image.clean_img(fmri_img)` to `image.clean_img(fmri_img, standardize=False)`?